ably/proto/crypto_test.go

```diff
@@ -29,3 +30,66 @@ func TestGenerateRandomKey(t *testing.T) {
 		}
 	})
 }
+
+func TestRealtimeChannelModes_ToFlag(t *testing.T) {
```

Comment:
Not sure, where to put this stuff, I might just refactor other stuff too that's part of channel options

Reply:
resolved, created new file `channel_options` and `channel_options_test` for channel options, lets not combine them with 
crypto, since we will have more channel options in the future